### PR TITLE
FF8: world debug options

### PR DESCRIPTION
**Pull request by MaKiPL**
Added imgui world debug section for worldmap

Features:

- camera zoom
- player position
- collision disable/enable
- current colliding triangle full debug and editing including flags

### CODE REVIEW

File: src/world.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include <imgui.h>
```
Comment (julianxhokaxhiu) [inline src/world.cpp:1]: Please remember to add the license header too here with your name :) Thank you!

File: src/world.h [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/****************************************************************************/
+//    Copyright (C) 2009 Aali132                                            //
+//    Copyright (C) 2018 quantumpencil                                      //
+//    Copyright (C) 2018 Maxime Bacoux                                      //
+//    Copyright (C) 2020 Julian Xhokaxhiu                                   //
+//    Copyright (C) 2020 myst6re                                            //
+//    Copyright (C) 2020 Chris Rizzitello                                   //
+//    Copyright (C) 2020 John Pritchard                                     //
+//    Copyright (C) 2020 Marcin Gomulak                                     //
```
Comment (julianxhokaxhiu) [inline src/world.h:9]: As you'll be part of this project starting this PR, please ensure your name is present everywhere this header is. I'll work on streamlining the injection of it in the source files somehow via CMake. Until then, you'd have to handle this manually. Painful process, I know...

**Comment by MaKiPL** [comment]
Github says there's option to automatically squash the commits- I broke something, probably due to merging with current master

**Comment by julianxhokaxhiu** [comment]
Fantastic work! Thank you :)

**Pull request merged**